runtime/vm/ValueTypeHelpers.cpp

```diff
@@ -250,4 +250,15 @@ areValueTypesEnabled(J9JavaVM *vm)
 #endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
 }
 
+j9object_t
+getDefaultValue(J9Class* clz)
+{
+	Assert_VM_notNull(clz);
```

Comment:
You don't have to assert clz is not NULL. If it is null, it will crash at the following line, which is effectively the same as the assertion.